### PR TITLE
Handle the issue on Commission management

### PR DESCRIPTION
**Pull request by RamyaVienna**

### CODE REVIEW

File: ModelLibrary/Model/MCommission.cs [hunk 1]
```diff
@@ -65,7 +65,7 @@ public MCommission(Ctx ctx, DataRow dr, Trx trxName)
         /// <returns>array of lines</returns>
         public MCommissionLine[] GetLines()
         {
-            String sql = "SELECT * FROM C_CommissionLine WHERE C_Commission_ID=@comid ORDER BY M_Product_ID, M_Product_Category_ID,C_BPartner_ID,C_BP_Group_ID,AD_Org_ID,C_SalesRegion_ID";
+            String sql = "SELECT * FROM C_CommissionLine WHERE C_Commission_ID=@comid AND ISActive='Y' ORDER BY M_Product_ID, M_Product_Category_ID,C_BPartner_ID,C_BP_Group_ID,AD_Org_ID,C_SalesRegion_ID";
```
Comment (Bharat-Singla) [inline ModelLibrary/Model/MCommission.cs:68]: check Case in IsActive

File: ModelLibrary/Process/CommissionCalc.cs [hunk 2]
```diff
@@ -84,6 +84,11 @@ protected override String DoIt()
                 throw new Exception("Could not save Commission Run please check Organization");
 
             MCommissionLine[] lines = m_com.GetLines();
+            //should not generate commission if there is no active commission lines
+            if (lines.Length <= 0)
+            {
+                throw new Exception("No Active Records Found");
```
Comment (Bharat-Singla) [inline ModelLibrary/Process/CommissionCalc.cs:90]: do not fix the message, get it from Message window

File: ModelLibrary/Process/CommissionCalc.cs [hunk 3]
```diff
@@ -149,7 +154,7 @@ protected override String DoIt()
                     {
                         sql.Append("SELECT h.C_Currency_ID, SUM(l.LineNetAmt) AS Amt,"
                             + " SUM(l.QtyOrdered) AS Qty, "
-                            + "NULL, NULL, NULL, NULL, MAX(h.DateOrdered) "
+                            + "l.C_orderline_ID, NULL, NULL, NULL, MAX(h.DateOrdered) "
```
Comment (Bharat-Singla) [inline ModelLibrary/Process/CommissionCalc.cs:157]: check case of C_OrderLine_ID

**Review by Bharat-Vienna — APPROVED** [review]
OK

**Pull request merged**